lua/lspconfig/lean3ls.lua

```diff
@@ -28,6 +28,10 @@ Once Lean is installed, you can install the Lean 3 language server by running
 ```sh
 npm install -g lean-language-server
 ```
+
+Note: that if you're using [lean.nvim](https://github.com/Julian/lean.nvim),
+that plugin fully handles the setup of the Lean language server,
+and you shouldn't set up `lean3ls` both with it and this plugin.
```

Comment:
maybe this plugin -> lspconfig just to be more explicit

Reply:
Done, thanks (as well as for `nvim-metals`, which I copied this from :D)